test/unit/features/transition/transition-mode.spec.js

```diff
@@ -457,7 +457,7 @@ if (!isIE9) {
             setTimeout(() => {
               resolve({ template: '<div><h1>component B</h1></div>' })
               Vue.nextTick(next)
-            }, (duration + buffer) * 1.5)
+            }, (duration + buffer) * 1.7)
```

Comment:
I was noticing timing issues while running the vue tests on my laptop locally. This test would pass if part of the entire suite, but if it was marked as a focused test, it would fail due to timing issues. I encountered this with the code prior to my change as well